pulsar/consumer_regex.go

```diff
@@ -383,11 +382,14 @@ func subscriber(c *client, topics []string, opts ConsumerOptions, ch chan Consum
 	return consumerErrorCh
 }
 
-func filterTopics(topics []string, regex *regexp.Regexp) []string {
+func filterTopics(topics []string, regex *regexp.Regexp) ([]string, error) {
 	matches := make(map[string]bool)
 	matching := make([]string, 0)
 	for _, t := range topics {
-		tn, _ := internal.ParseTopicName(t)
+		tn, err := internal.ParseTopicName(t)
```

Comment:
I'm not sure if this good thing to error out here. The list of topic passed here usually comes from a response from the broker, right?

Reply:
when c.client.namespaceTopics() return array length is 0 and err id nil.Does this situation exist?

Reply:
I'm not sure too.Although this is returned by the broker.